doc/attr.xml

```diff
@@ -2399,3 +2399,29 @@ gap> Length(M);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="Digraph2Kings">
+<ManSection>
+  <Oper Name="Digraph2Kings" Arg="D"/>
+  <Returns>A list.</Returns>
+  <Description>
+    If <A>D</A> is a tournament then this operation returns a list of the
+    2-kings in the tournament. If the tournament contains a source, then the
+    source is the only 2-king (see <Ref Attr="DigraphSources"/>). The number 
+    of 2-kings in a tournament without a source is at least three. A tournament
+    cannot have exactly two 2-kings.
```

Comment:
You could also please provide a cross reference in this manual entry to `DigraphIsKing`, so that a reader who doesn't know what a 2-king is can easily access that information 🙂